socorro/processor/mozilla_processor_2015.py

```diff
@@ -63,6 +63,7 @@
         "socorro.processor.signature_utilities.SignatureRunWatchDog, "
         "socorro.processor.signature_utilities.SignatureIPCChannelError, "
         "socorro.processor.signature_utilities.SigTrunc, "
+        "socorro.processor.signature_utilities.SignatureIPCMessageName, "
```

Comment:
I think this should not be after the ``SigTrunc`` rule, because that's the one that enforce signatures cannot be longer than some specific size. Please move it up one line.